README.md

```diff
@@ -77,6 +77,7 @@ Password #2, while not allowed by our policy, is only susceptible to a brute for
         - Spanish (es)
         - Telugu (te)
         - Ukrainian (uk)
+        - Chinaese (zh)
```

Comment:
This seems to be misspelled.

Reply:
Sorry, it is indeed a spelling mistake. It should be Chinese.